analyzer/src/main/kotlin/AnalyzerResultBuilder.kt

```diff
@@ -46,9 +46,18 @@ class AnalyzerResultBuilder {
 
     fun build(excludes: Excludes = Excludes.EMPTY): AnalyzerResult {
         val duplicates = (projects.map { it.toPackage() } + packages).getDuplicates { it.id }
-        require(duplicates.isEmpty()) {
+
```

Comment:
I've got some story maybe worth sharing here: I recently scanned ORT with ORT locally, 
it exited complaining in about duplicates in this function. I debugged it only quick and had to stop at some point due to other tasks. If I didn't make a mistake the finding was that there were multiple packages (not projects) with the same `id` which were completely identical except for on attribute. IIRC it was the homepage URL.

That's just FYI, didn't look into this PR deeply yet .

Reply:
I also had the case of multiple packages with the same identifier. This happened when a package was referenced from multiple package managers using the same package type (for instance _GoMod_ and _GoDep_). If the package metadata is not exactly the same, multiple `Package` instances are created.

Reply:
In my above case the full identifiers were identical, including the `type`.